candybot/candybot/commands/admin/blacklist.py

```diff
@@ -4,18 +4,24 @@
 
 class BlacklistCommand(AdminCommand):
     name = "blacklist"
-    help = "Blacklists a user from interacting with CandyBot."
+    help = "Shows current blacklist or blacklists a user from interacting with CandyBot."
     aliases = ["bl"]
-    examples = ["@User", "User#1234", "123456789"]
+    examples = ["", "@User", "User#1234", "123456789"]
     argument_spec = ArgumentSpec([UserArgument], False)
     clean = True
     ignore = False
 
+    title = ":lock: CandyBot Blacklist"
+    
     async def _run(self):
-        blacklist = database.get_blacklist(self.message.guild.id)
-        if self.user.id in blacklist:
-            database.set_blacklist(self.message.guild.id, self.user.id, remove=True)
-            await self.send(f"{self.user.mention} was removed from the CandyBot blacklist")
+        blacklist = database.get_blacklist()
+        if self.user is None:
+            # How to get user mentions from DB?
```

Comment:
See the leaderboard command for how to do this. You already have a list of user ids in `blacklist`, just need to convert them to `@mentions` using the `discord` interface.